aim/ext/tensorboard_tracker/tracker.py

```diff
@@ -10,10 +10,8 @@
 import weakref
 import queue
 
-from typing import TYPE_CHECKING, Any
-
-if TYPE_CHECKING:
```

Comment:
Was there a reason why these were only being imported for type checking?

Reply:
no, checked the code, just a mistake I guess